color/p5js/Law of simmilarity + color/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: [p5js]
+author: [Daniele Maselli]
+author-url: https://github.com/DanieleMaselli
+title: color + similarity
+tags: [P5.js, color, "Law of "]
```

Comment:
Law of what? please add in the Law you think that is right and don't put it into quotes.   

Wrong:    

    "Law of Similarity"  

Right:  

    Law of Similarity